jenkins/automation/alex.groovy

```diff
@@ -371,16 +386,28 @@ pipeline {
                 script{
                     withCredentials([usernamePassword(credentialsId: 'cortx-admin-github', usernameVariable: 'USERNAME', passwordVariable: 'PASSWORD')]) {
 						sh '''
-							pushd ./cortx/metrics/pickles
+							#pushd ./cortx
+							#pushd ./alex-test
+							git clone https://github.com/Seagate/alex-test.git
+							pushd ./alex-test
+							cp -r /var/tmp/alex.pickle .
 							git add alex.pickle
-							git commit -m "Update the Alex pickle"
-							git push https://${USERNAME}:${PASSWORD}@github.com/seagate/cortx.git HEAD:main
+							git commit -m "Alex pickle update"
```

Comment:
should we add `-s` option as well to sign commit?

Reply:
This pickle file we are adding in the alex-test repository. So can you please confirm if this is required, then will add it.

Reply:
@subhalaxmisa I guess adding pickle in `alext-test` is not helpful. As we are not using `alex-test` to generate reports. If we are planning to use then we also need to keep `main` and `alex-test` in sync which will be an additional overhead. 
As discussed pickel file for Alex should be dynamically generated whenever the Jupyter notebook is executed.

Reply:
Agreed. Let's discuss about this more in details. For monthly report we don't need any pickle file. So are we good to remove pickle related tasks?

Reply:
Yes. Please

Reply:
@shailesh-vaidya Done, since this is a prod job hence changed the default values as well and added one repository.

https://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/Subha_WS/job/Alex-scan/66/console